docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.4"
 
 services:
-  localstack-s3:
+  localstack:
```

Comment:
I'm surprised there isn't a standard way to setup buckets into `localstack`, either through its internal CLI tools or via configuration variables

Reply:
we do something similar to this PR in https://github.com/elifesciences/personalised-covers-formula/blob/a15351661fd2935dfd001bd679632e09601556b8/salt/personalised-covers/localstack.sls#L3

Reply:
I don't see anything other than using the aws cli.